### PR TITLE
alerting docs: slack integration

### PR DESCRIPTION
**Pull request by tonypowa**
**What is this feature?**

This is new documentation to help users connect Grafana and Slack

**Why do we need this feature?**

Users had to visit external websites to learn how to integrate Grafana and Slack 

**Who is this feature for?**

Any Grafana user

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Slack
+title: Configure Slack for Alerting
+weight: 300
+---
+
+
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:19]:
```suggestion
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+## Configure Slack for Alerting
+
+
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:22]:
```suggestion
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Use the Grafana Alerting - Slack integration to send Slack notifications when your alerts are firing. 
+
+There are two ways of integrating Slack into Grafana Alerting. 
+
+
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:27]:
```suggestion
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+---
+
+
+## Configure Slack for Alerting
+
+
+Use the Grafana Alerting - Slack integration to send Slack notifications when your alerts are firing. 
+
+There are two ways of integrating Slack into Grafana Alerting. 
+
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:26]:
```suggestion
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+1. Use a [Slack API token](https://api.slack.com/authentication/token-types)
+Enable your app to access the Slack API. If, for example, you are interested in more granular control over permissions, or your project is expected to regularly scale, resulting in new channels being created, this is the best option. 
+
+
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:31]:
```suggestion
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+1. Navigate to Alerts&IRM ->  Alerting -> Contact points.
+1. Click + Add contact point.
+1. Enter a contact point name.
+1. From the Integration list, select Slack.
+1. If you are using a Slack API token:
+- In the Recipient field, copy in the channel ID in the Recipient field.
+- In the Token field, copy in the Bot User OAuth Token that starts with “xoxb-”.
+6. If you are using a Webhook URL, in the Webhook field, copy in your Slack app Webhook URL.
+7. Click Test to check that your integration works.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:87]:
```suggestion
7. Click **Test** to check that your integration works. 
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+1. Navigate to Alerts&IRM -> Alerting -> Notification policies.
+1. In the Default policy, click  the ellipsis icon (…) and then Edit,
+1. Change the default policy to the contact point you created.
+1. Click Update default policy. 
+
+**Note:**
+
+{{< admonition type="note" >}}
+If you have more than one contact point, add a new notification policy rather than edit the default one, so you can route specific alerts to Slack. For more information, refer to Notification policies - [here](https://grafana.com/docs/grafana/latest/alerting/alerting-rules/create-notification-policy/#add-new-nested-policy).
+{{< /admonition >}}
```
[the remark this answers is not in the document]
Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:103]: i see! . And does it apply to external links (such as the links to Slack )?

[the remark this answers is not in the document]
Comment (brendamuir) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:103]: nope just internal ones

[the remark this answers is not in the document]
Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:85]: ✔️  hope i got it right

**Review by brendamuir — APPROVED** [review]

**Pull request merged**